server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/GenericTransportContext.java

```diff
@@ -13,26 +13,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-package org.kaaproject.kaa.server.bootstrap.service.http;
-
-import org.kaaproject.kaa.server.common.server.http.DefaultHttpServerInitializer;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+package org.kaaproject.kaa.server.transport;
 
 /**
- * The Class KaaHttpServerInitializer.
+ * A context for transport initialization parameters with binary configuration.
```

Comment:
Start a class description with a verb, like "Provides the context for the transport initialization parameters with binary configuration."